misago/users/management/commands/removeoldips.py

```diff
@@ -13,6 +13,6 @@ def handle(self, *args, **options):
         self.stdout.write("IP anonymization is disabled.")
         return
       
-      anonymize_old_ips.send(sender=self)
+      remove_old_ips.send(sender=self)
       self.stdout.write("IP addresses older than {} days have been anonymized.".format(settings.MISAGO_IP_STORE_TIME))
```

Comment:
Change this line to `self.stdout.write("IP addresses older than {} days have been removed.".format(settings.MISAGO_IP_STORE_TIME))`